### PR TITLE
[Documentation] Issue tracker help markdown

### LINKED ISSUE #6565: [Issue Tracker] Missing Help on Issue Add/Edit
If you navigate on https://test-loris-dev.loris.ca/issue_tracker/issue/42
And select the help option, it displays:
No help available for issue (/var/www/Loris/modules/issue_tracker/help/issue.md)

The same behaviour occurs if you create a New Issue.

### PR DESCRIPTION
**Pull request by laemtl**
I created a new markdown file (modules/issue_tracker/help/issue.md) to add help text on Issue Tracker > Create/Edit Issue

* Resolves #6565

### CODE REVIEW

File: modules/issue_tracker/help/issue.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Issue Tracker: Add or Edit Issue
+
+On this page, you can submit and edit issues.
+Then, populate the fields and click **Submit Issue**.
```
Comment (christinerogers) [inline modules/issue_tracker/help/issue.md:4]:
```suggestion
Populate or modify the fields in this page and click **Submit Issue**. 
```

**Comment by laemtl** [comment]
Thanks @christinerogers for your comment. I updated the screenshot in the description with the new version.

**Comment by christinerogers** [comment]
ready for final review @driusan

File: modules/issue_tracker/help/issue.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Issue Tracker: Add or Edit Issue
+
+On this page, you can submit and edit issues. Populate or modify the fields in this page and click **Submit Issue**. Note that you cannot submit PSCIDs from sites you don't have access to.
+
+## Watching list ##
```
Comment (driusan) [inline modules/issue_tracker/help/issue.md:5]: does this render correctly with the `##` at the end? It looks like a typo.

**Pull request merged**